<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ticket_page</class>
 <widget class="QDialog" name="ticket_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart cinema</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix3/beige.jpg);</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>891</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/new/prefix3/orange.png);</string>
   </property>
   <property name="title">
    <string>Gestion ticket</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>771</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/new/prefix2/marron.jpg);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Ajout</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>111</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>240</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Consommation</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>10</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Film</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_id">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_type">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>90</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_consommation">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_film">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>10</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>90</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Cine</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_cine">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>90</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>170</y>
        <width>91</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Salle</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_salle">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>170</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_film_cine">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>80</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_cine_salle">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>90</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_salle_horaire">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>170</y>
        <width>81</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm_type">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>80</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm_ajout">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>310</y>
        <width>171</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Affichage</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_confirm_aff_ticket">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>320</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Afficher</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_aff_ticket">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>40</y>
        <width>651</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Suppression</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_delete">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>60</y>
        <width>371</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
      <property name="text">
       <string>Enter ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confirm">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Modificaton</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_id_update">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>50</y>
        <width>291</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
      <property name="text">
       <string>Insert ID</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_verif_id_update">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>230</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>310</y>
        <width>191</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Consommation</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_update_type">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>153</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_update_cons">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>310</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image: whit</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_price_update">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>230</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: whit</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_update_confi_type">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>150</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_confi_update">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>310</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius : 10px ;
color: rgb(0, 0, 127);
font: 14pt &quot;MS Shell Dlg 2&quot;;
border-image: url(:/new/prefix3/orange.png);</string>
      </property>
      <property name="text">
       <string>Confirm update</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
